pkg/actions/mustgather.go

```diff
@@ -0,0 +1,315 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package actions
+
+import (
+	"archive/tar"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/eclipse/codewind-installer/pkg/docker"
+	"github.com/eclipse/codewind-installer/pkg/errors"
+	"github.com/eclipse/codewind-installer/pkg/utils"
+	"github.com/urfave/cli"
+)
+
+var codewindHome = filepath.Join(homeDir, ".codewind")
+var nowTime = time.Now().Format("20060102150405")
+var mustGatherMasterDirName = filepath.Join(codewindHome, "mustgather")
+var mustGatherDirName = filepath.Join(mustGatherMasterDirName, nowTime)
+
+var isLoud = true
+
+func logMG(input ...string) {
```

Comment:
This function is a bit odd, it takes any number of strings as an argument *but* only prints out the first one. All the callers only ever call it with one argument. Could this be simplified down to:
```
func logMG(input string) {
	if isLoud {
		fmt.Println(input)
	}
}
```
?

Reply:
Agreed